PowerEditor/src/NppCommands.cpp

```diff
@@ -105,6 +105,15 @@ void Notepad_plus::command(int id)
 		}
 		break;
 		
+		case IDM_FILE_OPEN_DEFAULT_VIEWER:
+		{
+			// Opens file in its default viewer. 
+            // Has the same effect as double–clicking this file in Windows Explorer.
+            BufferID buf = _pEditView->getCurrentBufferID();
+			ShellExecute(NULL, NULL, buf->getFullPathName(), NULL, NULL, SW_SHOW);

```

Comment:
Hi,
If there is no association for a file type, widows will ask with it program to open it.
How do you open this file? (Comment voulez-vous ouvrir ce fichier ?)
![windowsask](https://user-images.githubusercontent.com/12849364/28794343-45936928-7636-11e7-8b02-be25a277078a.jpg)